### PR TITLE
Email billing schedule after suspend

### PR DESCRIPTION
**Pull request by paulbrown1982**
- Send the ExactTarget data extension JSON onto an SQS queue, using the new HolidaySuspensionBillingScheduleDataExtensionRow for formatting.
- Separate out building of data extension row from the enqueuing of the message.
- Use the new getByContactId and BillingSchedule.rollup from membership-common-0.255

cc @tomverran @mario-galic

### CODE REVIEW

File: app/controllers/AccountManagement.scala [hunk 1]
```diff
@@ -118,15 +118,22 @@ object AccountManagement extends Controller with LazyLogging {
       pendingHolidays = pendingHolidayRefunds(newHolidays)
```
Comment (tomverran) [inline app/controllers/AccountManagement.scala]: This wasn't your code but there's a call to getHolidays in this file that returnsString \/ Seq[Holiday] where the string is going to be some Zuora JSON we failed to deserialise, and it'll be printed out in BadRequest unless we leftMap it to something like Internal Error first

File: app/model/exactTarget/DataExtensionRow.scala [hunk 2]
```diff
@@ -163,10 +167,60 @@ object PaperHomeDeliveryWelcome1DataExtensionRow {
...
+           packageName: String,
+           billingSchedule: BillingSchedule,
+           numberOfSuspensionsLinedUp: Int,
+           daysUsed: Int,
+           daysAllowed: Int
+         ): HolidaySuspensionBillingScheduleDataExtensionRow = {
+
+    val (thereafterBill, trimmedSchedule) = BillingSchedule.rolledUp(billingSchedule)
+    val count = new AtomicInteger()
+    val futureBills = trimmedSchedule.flatMap(bill => {
```
Comment (tomverran) [inline app/model/exactTarget/DataExtensionRow.scala]: You may find zipWithIndex helpful here - it takes a list[A] and maps it to a List[(Int, A)] where the Int is the index

Comment (paulbrown1982) [inline app/model/exactTarget/DataExtensionRow.scala]: Ok I'll fix that when I rebase https://github.com/guardian/subscriptions-frontend/pull/547 to master and resubmit it.

File: app/controllers/AccountManagement.scala (hunk 1 above)
Comment (paulbrown1982) [inline app/controllers/AccountManagement.scala]: Ok I'll fix that when I rebase https://github.com/guardian/subscriptions-frontend/pull/547 to master and resubmit it.

**Comment by tomverran** [comment]
👍

**Pull request merged**